tests/deltatuner/finetune/run_denas_ssf.sh

```diff
@@ -0,0 +1,47 @@
+set -x
+
+DATA_PATH="/home/vmagent/app/data"
+LOG_PATH=$DATA_PATH"/dtuner_test/log"
+MODEL_SAVE_PATH=$DATA_PATH"/dtuner_test/models"
+
```

Comment:
This is the example scripts, not cicd , suggest to split mpt and llama finetuning scripts for convenient experiments